rero_ils/modules/patrons/jsonschemas/form_patrons/patron-v0.0.1.json

```diff
@@ -74,13 +74,23 @@
           {
             "key": "roles",
             "notitle": "true",
-            "description": "Is the user a patron or/and a librarian?",
+            "description": "User role(s) ?",
             "type": "checkboxes",
             "htmlClass": "d-inline-block",
             "orderable": "false",
-            "enum": [
-              "patron",
-              "librarian"
+            "titleMap": [
+              {
+                "value": "patron",
+                "name": "patron"
+              },
+              {
+                "value": "librarian",
+                "name": "librarian"
+              },
+              {
+                "value": "systemlibrarian",
+                "name": "systemlibrarian"
```

Comment:
`"name": "system librarian"`

Else, why a a titlemap if the values are all equal names?

Reply:
To allow translation